check_springboot_actuator.py

```diff
@@ -29,6 +29,10 @@
     '-u', '--user-credentials',
     help='user credentials in the format username:password',
     dest='credentials')
+helper.parser.add_option(
+    '-T', '--timeout',
```

Comment:
The parameter "timeout" sshould be renamed to timeout - otherwise this error occurs:
`optparse.OptionConflictError: option -o/--timeout: conflicting option string(s): --timeout`

Reply:
I pushed this as a new commit